### DISCUSSION TITLE
Prot Pala updated in pre patch?

### DISCUSSION THREAD

**Discussion by Kaiserlow** [Hekili/hekili#1695] [General]
Hey there. Noob here. Is prot pala updatet 

The spells it shows are not correct at all and my add on shows that it has not been updated yet - see screenshot.
Am I missing something (me being dumb) or is it just not updated yet?

greetings 
![prot](https://user-images.githubusercontent.com/116746642/198078363-5a9bd66a-0997-4a4d-8395-4f6a4d884d8f.jpg)

**Comment by jmusick** [comment]
1- The APL probably hasn't been updated by SimC devs yet for Hekili to import, some of them may not be updated until Dragonflight proper releases either
2- You'll need to select beta releases in CurseForge to get the most recent updates that are coming out over the past 24 hours or so. That doesn't mean it'll have an update for your preferred class/spec right away

**Reply by Kaiserlow** [reply]
Thanks m8!

**Comment by Hekili** [comment]
Many specs do have priority lists (or they are works-in-progress) and I will include them.  I'm working through it ASAP.  In the next beta, the addon will simply not provide info for specs I haven't finished updating.

**Reply by Kaiserlow** [reply]
Great job, right on!
